lib/RoadizCoreBundle/src/Routing/NodeRouteHelper.php

```diff
@@ -28,7 +28,7 @@ public function __construct(
         private readonly PreviewResolverInterface $previewResolver,
         private readonly LoggerInterface $logger,
         private readonly string $defaultControllerClass,
-        private readonly NodeTypes $nodeTypesBag,
+        private readonly DecoratedNodeTypes $nodeTypesBag,
```

Comment:
Same here, NodeRouteHelper is not displaying any information to back-office.
You should stick to regular NodeTypes